content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,111 @@
+
+- Download and install [Docker Desktop version 4.25](release-notes.md)
+- Have a Pro, Team, or Business subscription
+- Have a Docker ID that's part of a Docker organization participating in the Early access program
+
+To enable Synchronized file sharing:
+1. Sign in to Docker Desktop.
+2. Anywhere within the Docker Dashboard, enter the Konami Code which has the key sequence of `up, up, down, down, left, right, left, b, a`.
+3. From the list of experimental features, select **Mutagen File Sharing**.
+4. At the bottom of the list, select **Go back to dashboard**
+
+Synchronized file sharing is now enabled and ready for you to create your first file share instance.
+
+## Create a file share instance 
+
+To create a synchronized file share:
+1. Navigate to **Settings** within Docker Desktop and then within the **Resource** section, select the **File sharing** tab. 
+2. In the **Synchronized file shares** section, select the **plus** icon.
+3. Select a host folder to share. The synchronized file share should initialize and be usable.
+
+File shares take a few seconds to initialize as files are copied into the Docker Desktop VM. During this time, the status indicator displays **Preparing**.
+
+When the status indicator displays **Watching for filesystem changes**, your files are available to the VM through all the standard bind mount mechanisms, whether that's `-v` in the command line or specified in your `compose.yml` file.
+
+>**Note**
+>
+> When you create a new service, setting the [bind mount option consistency](../engine/reference/commandline/secret_create.md#options-for-bind-mounts) to `:consistent` bypasses synchronized file sharing. 
+
+## Explore your file share instance
+
+The **Synchronized file shares** section displays all your file share instances and provides useful information about each instance including:
+- The origin of the file share content
+- A Status update
+- How much space each file share is using
+- The number of filesystem entry counts
+- The number of symbolic links
+- Which container(s) is using the file share instance
+
+Selecting a file share instance expands the dropdown and exposes this information.
+
+## Use `.syncignore`
+
+You can use a `.syncignore` file at the root of each file share, to exclude local files from your file share instance. It supports the same syntax as `.dockerignore` files and excludes, and/or re-includes, paths from synchronization. `.syncignore` files are ignored at any location other than the root of the file share.
+ 
+Some example of things you might want to add to your `.syncignore` file are:
+- Large dependency directories, for example `node_modules` and `composer` directories (unless you rely on accessing them via a bind mount)
+- `.git directories` (again, unless you need them)
+
+In general, the contents of your `.syncignore` file can include things anything that's going to cost you time to sync up-front and cost you storage. 
```

Comment:
There's an extra "things" here that I think is accidental.

I'd also maybe phrase it along the lines of "In general, your `.syncignore` file should be used to exclude items that aren't critical to your workflow, especially those that would be slow to sync or use significant storage."

I worry the term "up-front" my falsely imply some sort of deferred syncing.

Reply:
Done, changed